WebRoot/js/zimbraMail/abook/view/ZmContactSplitView.js

```diff
@@ -1036,7 +1036,7 @@ ZmContactSimpleView = function(params) {
 	params.className = "ZmContactSimpleView";
 	ZmContactsBaseView.call(this, params);
 
-	this._normalClass = DwtListView.ROW_CLASS + " SimpleContact";
+	this._normalClass = "SimpleContact " + DwtListView.ROW_CLASS;
```

Comment:
It does matter. DragProxy logic appends "-dragProxy" tot he end of the _normalClass variable. Updated the order to have it consistent class name across the apps. (Row-dragProxy/RowDouble-dragProxy).